.github/workflows/release.yaml

```diff
@@ -164,3 +157,19 @@ jobs:
         with:
           name: artifacts
           path: dist/**/*
+
+  release-docker-assets:
```

Comment:
not a blocking comment: if this fails we'd need to republish the entire release. We could change this section to a post-release workflow:
```
# .github/workflows/publish-docker.yaml

name: "Release follow-up"
on:
  release:
    types: [published]
  workflow_dispatch:
    inputs:
      tag:
        description: 'The tag to base publication from'
        required: true 
        type: string

jobs:
 ...same...

```
In which case, if the release succeeds but the docker publish fails then we could modify and re-run the publishing in a workflow-dispatch (manual) fashion (thus, we wouldn't need to re-tag and re-release).

Reply:
I think we're actually able to re-run this as a JOB on the actions page if it fails. So if `quality-gate` and `release` pass, but `release-docker-assets` fails we have to option to re-run all failed jobs. which in this case would just be the `release-docker-assets` job.

Reply:
I think this makes sense, as long as the failure doesn't require a code update 👍